MUSHclient/worlds/plugins/aard_channels_fiendish.xml

```diff
@@ -190,6 +190,18 @@ if window_title == "" then
    window_title = nil
 end
 
+function clearTab(tab)
+   if tab == nil then
+      tab = current_tab
+   elseif type(tab) == "string" then
+      tab = findTab(tab)
+   end
+   tabs_rects[tab]:clear(true)
```

Comment:
If the tab isn't found this will error

Reply:
That makes sense. I also changed to current_tab or 1 in the event that current_tab, for whatever reason, isn't loaded.